.changeset/little-roses-kick.md

```diff
@@ -0,0 +1,14 @@
+---
+"@jspsych/plugin-canvas-keyboard-response": patch
+"@jspsych/plugin-image-keyboard-response": patch
+"@jspsych/plugin-canvas-button-response": patch
+"@jspsych/plugin-canvas-slider-response": patch
+"@jspsych/plugin-image-button-response": patch
+"@jspsych/plugin-image-slider-response": patch
+"@jspsych/plugin-categorize-animation": patch
+"@jspsych/plugin-animation": patch
+"@jspsych/plugin-sketchpad": patch
+"@jspsych/config": patch
```

Comment:
Technically, this is a breaking change to the config package. I think it's unlikely that people have relied on the canvas dependency, but that doesn't change the fact it might be breaking for some folks :melting_face: I'll let you decide if we purposefully break the rules here with a patch change :sweat_smile: